src/orquestra/sdk/schema/ir.py

```diff
@@ -82,10 +82,11 @@ class InlineImport(BaseModel):
 
 
 class PackageSpec(BaseModel):
+    # noqa E51
```

Comment:
what's `E51`?

Reply:
It's E501 if you're really lazy!
```suggestion
    # noqa E501
```